releases/v1.24.0

```diff
@@ -0,0 +1,77 @@
+# Highlights
+
+## Custom Slot Supplier (Preview)
```

Comment:
@Sushisource Do these look good to you? I made some slight edits to your original notes

Reply:
I might change the heading to be "Slot Auto-Tuning" since that's the main add here, even though it is an extension to the custom slot suppliers.